tests/test_path.py

```diff
@@ -24,7 +24,10 @@ def test_path_extrude_simplify():
     P = pp.arc(radius=10, angle=90, num_pts=300)
     D = P.extrude(width=1, layer=3, simplify=1e-1)
     h = D.hash_geometry(precision=1e-4)
-    assert h == "92a41fb5afa37cde3b06b521ff14d8445d962069"
+    if sys.version_info >= (3, 8):
+        assert h == "eb36d790b478134fa14a08f43fe28f112839ab2d"
+    else:
+        assert h == "92a41fb5afa37cde3b06b521ff14d8445d962069"
```

Comment:
@amccaugh, I have added this check which makes the tests pass. The Polygons seem to be the same, it's just that the upstream hashes seem the have changed.